frontend/src/i18ns/en.js

```diff
@@ -43,5 +54,6 @@ export default {
         mustBeLessThan: "{minDmg} must be less than {maxDmg}",
         mustBePositive: "{allProps} must be positive",
         notValidType: "{allProps} not valid type",
+        notFound: "Page not found"
```

Comment:
We don't really have any pages, we have SPA app. I think "Resource is not found" is more appropriate.